survey_creation/de_17/invitation.txt

```diff
@@ -2,17 +2,17 @@ Umfrage zu Forschungssoftware / wissenschaftlicher Software in Deutschland und d
+Derzeit gibt es kaum zuverlässige Informationen über die Gemeinschaft der Software entwickelnden Personen in Forschung und Wissenschaft. Die Umfrage soll dabei helfen, Einblicke in und Informationen über diese Gemeinschaft zu gewinnen, um mit diesen Ergebnissen Förderorganisationen und Wissenschaftseinrichtungen bei der Entwicklung von Strategien und Förderprogrammen sowie von Leit- und Richtlinien unterstützen zu können.
 
-Die Umfrage gibt Dir die Möglichkeit Deine Sicht und Erfahrungen einfließen zu lassen und damit an der zukünftigen Gestaltung teilzuhaben.
+Die Umfrage gibt Dir die Möglichkeit, Deine Sicht und Erfahrung mit einfließen zu lassen und damit an der zukünftigen Gestaltung der Community teilzuhaben.
 
-Du kennst Andere, die Software im Umfeld von Forschung und Wissenschaft entwickeln. Dann lass sie bitte von dieser Umfrage wissen. Nur die rege Teilnahme und Antworten, die quer aus Deutschland und den unterschiedlichsten Bereichen kommen, gewährleisten einen umfassenden Eindruck zum Thema. Sollte diese E-Mail von Dir an Mailinglisten weitergeleitet werden, dann nehmt uns, Stephan und Martin, bitte ins CC, damit wir dies bei der Auswertung der Umfrage berücksichtigen können.
+Du kennst Andere, die Software im Umfeld von Forschung und Wissenschaft entwickeln? Dann informiere sie bitte über diese Umfrage! Nur eine rege Teilnahme und Antworten, die von überall aus Deutschland kommen und die unterschiedlichsten Bereiche repräsentieren, gewährleisten einen umfassenden Einblick zu diesem Thema. Sollte diese E-Mail von Dir an Mailinglisten weitergeleitet werden, dann nehmt uns, Stephan (*E-Mail-Adresse hier einfügen!*) und Martin (*E-Mail-Adresse hier einfügen!*), bitte ins Cc, damit wir dies bei der Auswertung der Umfrage berücksichtigen können.
 
-Die Ergebnisse werden unter einer CC BY-NC Lizenz veröffentlicht und zusätzlich auf der de-RSE Mailingliste [1] und dem de-RSE Blog [2] angekündigt und ausgewertet. Darüber hinaus werden ähnliche Umfragen in Großbritannien, Kanada, Australien, Norwegen, USA, Südafrika und Niederlande durchgeführt. Für einen besseren Vergleich, wurde die Gestaltung der deutschen Umfrage in enger Abstimmung zu diesen Umfragen durchgeführt.
+Die Ergebnisse werden unter einer CC-BY-NC Lizenz (*welche Version?*) veröffentlicht und zusätzlich auf der de-RSE Mailingliste [1] und dem de-RSE Blog [2] angekündigt und ausgewertet. Darüber hinaus werden ähnliche Umfragen in Großbritannien, Kanada, Australien, Norwegen, den USA, Südafrika und den Niederlanden durchgeführt. Um eine gute Vergleichbarkeit gewährleisten zu können wurde die Gestaltung der deutschen Umfrage in enger Abstimmung mit diesen Umfragen durchgeführt.
```

Comment:
Not sure whether you meant that the survey will actually be "angekündigt und ausgewertet" via the list and blog, or just that the results will be published via these channels? Same for English version...